components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -128,27 +146,58 @@ export default function () {
                             })}
                             {activeWorkspaces.length > 0 && <div className="py-6"></div>}
                             {inactiveWorkspaces.length > 0 && (
-                                <div className="p-3 text-gray-400 bg-gray-50 dark:bg-gray-800 rounded-xl text-sm text-center">
-                                    Unpinned workspaces that have been inactive for more than 14 days will be
-                                    automatically deleted.{" "}
-                                    <a
-                                        className="gp-link"
-                                        href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection"
+                                <div className="pt-14">
+                                    <div className="border-t border-gray-200 dark:border-gray-800"></div>
+                                    <div
+                                        onClick={() => setShowInactive(!showInactive)}
+                                        className="flex cursor-pointer py-6"
                                     >
-                                        Learn more
-                                    </a>
+                                        <div className="flex flex-col">
+                                            <h2 className="">Inactive Workspaces</h2>
```

Comment:
**suggestion(non-blocking)**: One more thing we could improve here is to make this element kind of dynamic so that when there are only inactive workspaces the dashboard still feel active, populated recent data, and ready to go.

| Approach :a: | Approach :b: |
|-|-|
| <img width="1440" alt="InactiveWorkspaces-t1" src="https://user-images.githubusercontent.com/120486/172188637-c3caf885-775f-4e39-8a79-90019925b46b.png"> | <img width="1440" alt="InactiveWorkspaces-t2" src="https://user-images.githubusercontent.com/120486/172188798-edff5d02-d540-4497-be4b-37b7dd9aa27b.png"> |

This information (amount of inactive workspaces) could be also nice-to-have inside the deletion confirmation modal.

| Modal using approach :a: | Modal using approach :b: |
|-|-|
| <img width="1440" alt="InactiveWorkspaces" src="https://user-images.githubusercontent.com/120486/172188977-e6f531fb-4ece-4e80-b5bb-64f2ce57903e.png"> | <img width="1440" alt="InactiveWorkspaces-1" src="https://user-images.githubusercontent.com/120486/172188984-e80fbf3c-6777-401a-b7f9-81812c0ed7d2.png"> |

Reply:
Marking as resolved. Moved forward with adding the workspace length in a parenthesis at the end of the section header. ✔️